clients/include/testing_gemm_strided_batched.hpp

```diff
@@ -22,18 +22,8 @@ void testing_gemm_strided_batched(const Arguments& arg)
     rocblas_int N = arg.N;
     rocblas_int K = arg.K;
 
-    T h_alpha;
-    T h_beta;
-    if(std::is_same<T, rocblas_half>{})
-    {
-        h_alpha = float_to_half(arg.alpha);
-        h_beta  = float_to_half(rocblas_isnan(arg.beta) ? 0 : arg.beta);
-    }
-    else
-    {
-        h_alpha = arg.alpha;
-        h_beta  = rocblas_isnan(arg.beta) ? 0 : arg.beta;
-    }
+    T h_alpha = arg.get_alpha<T>();
+    T h_beta  = rocblas_isnan(arg.get_beta<T>()) ? 0.0 : arg.get_beta<T>();
```

Comment:
`arg.get_beta<T>()` should already consider whether `rocblas_isnan(arg.beta)` and return accordingly.

Reply:
understand. This is my mistake. will update it on third commit "update complex change"